docs/UserGuide.md

```diff
@@ -47,7 +47,10 @@ An image of our UI is shown below!
         - [Delete](#deleting-an-entry)
```

Comment:
Would it be nicer if we put 'Deleting an entry: `edelete`' kind of format for our content list? From what I saw from others' user guides haha, but up to you to change